### PR TITLE
Cancel timeout on focus

### LINKED ISSUE #21: Feature Request: Disable Timeout on Focus
Toast is great, one feature that would be helpful is an option to disable the timeout feature on a notification when it has focus (hover or touch). This would allow visitors to focus on longer messages giving them time to read the whole thing without it disappearing. Maybe the option could be `hideOnFocus:` and then be set to `true` or `false`. 

For context, this was bought up within Nextcloud at https://github.com/nextcloud/server/pull/15124#pullrequestreview-227243101

### PR DESCRIPTION
**Pull request by skjnldsv**
Fix #21 

@apvarun

### CODE REVIEW

File: src/toastify.js [hunk 1]
```diff
@@ -116,6 +117,31 @@
           }.bind(this)
         );
 
+        // Reset timeout if toats is focused
```
Comment (raimund-schluessler) [inline src/toastify.js]: Should read "toast" 😉 Maybe you can rephrase it to "Clear timeout while toast is focused"? Seems more fitting to what the function does.

Otherwise it looks good.

**Comment by raimund-schluessler** [comment]
One drawback of adding the timeout again is that you can only stop one toast from disappearing at a time. But I guess this is fine (in the worst case you can move the cursor fast over all toasts 😄).

File: src/toastify.js (hunk 1 above)
Comment (skjnldsv) [reply src/toastify.js]: Indeed! :see_no_evil:

**Comment by skjnldsv** [comment]
Bump! :)
